tests/integration/src/test/java/org/mule/test/config/ConfigurationAnnotationsTestCase.java

```diff
@@ -83,8 +83,7 @@ public void testFlowAnnotations()
         assertThat(getDocDescription(flow), is("Main flow"));
         assertThat(getSourceFile(flow), is("annotations.xml"));
         assertThat(getSourceFileLine(flow), is(10));
-        // initialState is added by the xsd with a default value
-        assertThat(getSourceElement(flow), is("<flow name=\"Bridge\" doc:name=\"Bridge flow\" initialState=\"started\">" + SystemUtils.LINE_SEPARATOR +
+        assertThat(getSourceElement(flow), is("<flow name=\"Bridge\" doc:name=\"Bridge flow\">" + SystemUtils.LINE_SEPARATOR +
```

Comment:
why this change? seems like the only difference is that you removed the initialState attribute. WHat's the reason for this?